File: SIMINOV-HYBRID-PHONEGAP-TEMPLATE/assets/www/Liquor/Models/Credential.js
/** 
 * [SIMINOV FRAMEWORK]
 * Copyright [2015] [Siminov Software Solution LLP|support@siminov.com]
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 **/


function Credential() {

	//Class Variables.
	var accountId = null;
	var token = null;
	var active;

	this.getAccountId = function() {
		return accountId;
	}

	this.setAccountId = function(val) {
		accountId = val;
	}

	this.getToken = function() {
		return token;
	}

	this.setToken = function(val) {
		token = val;
	}

	this.getActive = function() {
		return active;
	}
	
	this.isActive = function() {
		return active;
	}

	this.setActive = function(val) {
		active = val;
	}
}


//Table Name
Credential.TABLE_NAME = "CREDENTIAL";


//Column Names
Credential.ACCOUNT_ID = "ACCOUNT_ID";
Credential.TOKEN = "TOKEN";
Credential.ACTIVE = "ACTIVE";


Function.extend(Database, Credential);

File: Hybrid-Phonegap-Library-Sample/Android/HybridLibrarySamples/src/main/java/siminov/hybrid/phonegap/library/sample/model/Credential.java
/** 
 * [SIMINOV FRAMEWORK]
 * Copyright [2015] [Siminov Software Solution LLP|support@siminov.com]
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 **/

package siminov.hybrid.phonegap.library.sample.model;

import siminov.core.database.Database;


public class Credential extends Database {

	//Column Names.
	public static final String ACCOUNT_ID = "ACCOUNT_ID";
	public static final String TOKEN = "TOKEN";
	public static final String ACTIVE = "ACTIVE";


	//Class Variables.
	private String accountId = null;
	private String token = null;
	private boolean active;

	public String getAccountId() {
		return this.accountId;
	}

	public void setAccountId(String accountId) {
		this.accountId = accountId;
	}

	public String getToken() {
		return this.token;
	}

	public void setToken(String token) {
		this.token = token;
	}

	public boolean getActive() {
		return this.active;
	}
	
	public boolean isActive() {
		return this.active;
	}

	public void setActive(boolean active) {
		this.active = active;
	}
}
